### PR TITLE
Add dependabot to automatically update dependencies

### PR DESCRIPTION
**Pull request by tunetheweb**
It would be nice to keep this tool up to date with latest version of SQLFluff.

It looks like it deploys automatically on merge to `master` so just adding a dependabot config file will cause GitHub to scan your requirements files and auto raise a Pull Request when a new version of any of the dependencies (including SQLFluff itself) is available.

This will save the manual effort of remembering to do this and, providing all tests pass, you can merge away and it should auto deploy.

### CODE REVIEW

**Review by nolanbconaway — APPROVED** [review]
Definitely, I meant to do something like this ages ago !

**Comment by tunetheweb** [comment]
Not sure if you're intending me to merge this now it's approved, or if you're going to come back to it, but looks like I don't have merge permissions in this repo.

Can you add the @sqlfluff/sqlfluff-maintainers as repo owners here like in the main repo?

**Review by barrywhart — APPROVED** [review]

**Comment by barrywhart** [comment]
Nice!

**Comment by nolanbconaway** [comment]
> Can you add the @sqlfluff/sqlfluff-maintainers as repo owners here like in the main repo?

good point! added as maintainers!

**Comment by tunetheweb** [comment]
Oops - maybe should have merged #28 first to get us up to date:

image

Ah well, they should all automatically close if you do merge that one. Except for last one (GitHub setup-python script) as didn't update that one.

**Comment by nolanbconaway** [comment]
pfffft

**Pull request merged**
